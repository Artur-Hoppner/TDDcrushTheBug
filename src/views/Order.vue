<template>
  <div></div>
</template>

<script>
export default {
  name: 'Order',
  components: {}
};
</script>

<style></style>
