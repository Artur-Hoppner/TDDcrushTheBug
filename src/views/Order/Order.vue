<template>
  <div>
    <OrderInfo />
  </div>
</template>

<script>
import OrderInfo from '@/components/OrderInfo/OrderInfo.vue';
export default {
  name: 'Order',
  components: {
    OrderInfo
  }
};
</script>

<style lang="scss" scoped></style>
