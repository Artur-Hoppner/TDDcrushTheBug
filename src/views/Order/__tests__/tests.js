import { shallowMount, createLocalVue } from '@vue/test-utils';
import Vuex from 'vuex';
import Order from '@/views/Order/Order.vue';

const localVue = createLocalVue();

localVue.use(Vuex);

describe('Test Order', () => {
  test('Test if /Order is rendered', () => {
    //Arrange
    const wrapper = shallowMount(Order, { localVue });
    //Assert
    expect(wrapper.element).toMatchSnapshot();
  });
});
