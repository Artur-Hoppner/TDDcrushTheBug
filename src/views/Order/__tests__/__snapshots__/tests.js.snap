// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Order Test if /Order is rendered 1`] = `
<div>
  <orderinfo-stub />
</div>
`;
